app/controllers/taxons_controller.rb

```diff
@@ -93,6 +103,10 @@ def destroy_flash_message(response_code)
     end
   end
 
+  def path_prefixes_for_select
+    Theme.taxon_path_prefixes
```

Comment:
Might be easier to put `Theme.taxon_path_prefixes` into the view directly, so avoid having to pass them around in the views.